src/charts/heatmap/heatmap.directive.js

```diff
@@ -271,8 +286,20 @@ angular.module('patternfly.charts').directive('pfHeatmap', function ($compile, $
 
         //Adding events
         blocks.on('mouseover', function () {
+          var fillColor;
           blocks.call(highlightBlock, false);
-          d3.select(this).call(highlightBlock, true);
+          if (scope.maxSize < 16) {
```

Comment:
FYI, I added the maxBlockHoverSize attribute to override the default value. However, I'm not using blockSize because that adds padding. When the maxBlockSize attribute is 15, blockSize equals 20. If you inspect this via the browser, the physical block is actually 16x16. In this case, blockSize is greater than the default and highlighting wouldn't be enabled.

This is more problematic when using the maxBlockHoverSize attribute to override the default value. For example, highlighting would be disabled when max-block-size="15" and max-block-hover-size="16" because blockSize is computed to be 20. I think this would be too confusing to someone trying to override these values.